src/main/java/me/contaria/seedqueue/gui/wall/SeedQueueWallScreen.java

```diff
@@ -803,6 +810,10 @@ public boolean isBenchmarking() {
         return this.benchmarkedSeeds < this.benchmarkGoal;
     }
 
+    public boolean shouldNotShowBenchmarkResults() {
+        return !this.showFinishedBenchmarkResults;
+    }
+
```

Comment:
Could you make `SeedQueueWallScreen#showFinishedBenchmarkResults` protected and access it directly in `SeedQueueBenchmarkToast`, similar to the other benchmark related fields?

Reply:
yes